### PR TITLE
Feature/feature flag on mobile app

### PR DESCRIPTION
**Pull request by bernarden**

### CODE REVIEW

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+﻿HOL - Feature Flag for a Mobile Solutions
+=========================================
+
+In this lab you will work with a simple Xamarin Forms mobile application. Your task is to add a feature flag and then bind it to the app's logic.
+
+## Pre-requisites: ##
+
+- Visual Studio 2015
+
+- [Xamarin](https://www.xamarin.com) installed and set up for Visual Studio.
```
Comment (steven2000nz) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Can this link be more specific - i.e. https://developer.xamarin.com/guides/cross-platform/getting_started/visual_studio_with_xamarin/

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+	![](media/8.png)
+
+* Add a "SwitchCell" tag into the "TableSection" tag in the following way:
+	```
+	...
+	<TableView Intent="Settings">
+		<TableSection Title="General">
+			<TextCell Text="Website" Detail="{Binding Website, Source={x:Static resources:GlobalResources.Current}}"/>
+			<SwitchCell On="{Binding IsFeatureFlagEnabled, Source={x:Static resources:GlobalResources.Current}}" Text="Feature flag"/>
```
Comment (steven2000nz) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Can we please this feature flag a more meaningful name - i.e. Extra Discount Feature Flag

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md (hunk 1 above)
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: done

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+* Right click on "PartsUnlimited.UWP" project, then on "Set as StartUp Project".
+
+	![](media/4.png)
+
+* Select the emulator you want to run this app on and click on green triangle to run the application.
+
+	![](media/5.png)
+
+* You should see the following main screen in your emulator:
```
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: done

File: docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
+﻿HOL - Feature Flag for a Mobile Solutions
+=========================================
+
+In this lab you will work with a simple Xamarin Forms mobile application. Your task is to add a feature flag and then bind it to the app's logic.
+
+## Pre-requisites: ##
```
Comment (bernarden) [inline docs/HOL-Feature_Flag_for_a_Mobile_Solutions/README.md]: Done

**Pull request closed without merging**